docs/production-deployment/cloud/audit-logging-aws.mdx

```diff
@@ -0,0 +1,138 @@
+---
+id: audit-logging-aws
+title: Audit Logging - AWS Kinesis
+sidebar_label: Amazon Kinesis
+description: Audit Logging in Temporal Cloud provides forensic information, integrating with AWS Kinesis for secure data handling and supporting key Admin and API Key operations. This streamlines audit and compliance processes.
+slug: /cloud/audit-logging-aws
+toc_max_heading_level: 4
+keywords:
+  - audit logging
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - troubleshooting
+  - aws
+  - kinesis
+tags:
+  - audit-logging
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - troubleshooting
+  - aws
+  - kinesis
+---
+
+## How to configure Audit Logging using AWS Kinesis {#configure-audit-logging}
```

Comment:
suggestion```
## Configure Audit Logging using AWS Kinesis {#configure-audit-logging}
**How to configure Audit Logging using AWS Kinesis**
```
We generally shouldn't have How to in our headers. Other than that, LGTM!